tests/drivers/can/api/twai-enable.overlay

```diff
@@ -4,6 +4,23 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/* Enable CAN bus */
 &twai {
-        status = "okay";
+	status = "okay";
+	pinctrl-0 = <&twai_default>;
+	pinctrl-names = "default";
+	bus-speed = <125000>;
+
+	can-transceiver {
+		max-bitrate = <1000000>;
+	};
+};
+
+&pinctrl {
+	twai_default: twai_default {
+		group1 {
+                        /* Create a virtual jumper connecting TWAI_TX and TWAI_RX */
+			pinmux = <ESP32_PINMUX(13, ESP_TWAI_RX, ESP_TWAI_TX)>;
```

Comment:
You can create an internal loopback using using the `output-enable` on input pins and `input-enable` on output pins, such as:
```suggestion
twai_default: twai_default {
		group1 {
			pinmux = <TWAI_RX_GPIO5>;
			output-enable; /* Enable internal loopback */
		};
		group2 {
			pinmux = <TWAI_TX_GPIO5>;
			input-enable; /* Enable internal loopback */
		};
```

Reply:
Ah, that was the one combination that I didn't try in my test that I listed in https://github.com/zephyrproject-rtos/zephyr/pull/59918#discussion_r1253918757!  I was thinking that the same GPIO pin would end up overwriting the GPIO mux, but this does work.  Will have to do another read of the code in pinctrl_esp32.c to make sure I understand it.

Reply:
I have updated the commit to use your code snippet.

Reply:
I'll mark this as resolved now that I've updated the commit.  Let me know if the etiquette is to have the initial commenter handle marking as resolved.